src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(y) {
+      if (y === undefined) {
+        this.coords.y++;
+      } else if (y > 0) {
+        this.coords.y += y;
+      }
```

Comment:
```suggestion
    goForward(value = 1) {
      if (value > 0) {
        this.coords.y += value;
      }
```
better use a default value for the parameter
fix it in all methods for robot movement

and better use 'value' or 'step' name for the variable to avoid confusing it with the coords property

Reply:
Cool, thanks! Got it and fixed it.